src/test/resources/features/Signin.feature

```diff
@@ -0,0 +1,4 @@
+Feature: Signin
+
+  Scenario: Salesforce sign in
+    Given I sign in as "rpfh1@mailinator.com" user and "654321*Aa"
```

Comment:
replace the user credentials by **username** and "password"

Reply:
#1 User credentials were removed from Signin.feature file